### DISCUSSION TITLE
Function problem tap_action: more-info

### DISCUSSION THREAD

**Discussion by stepsolar** [ExperienceLovelace/ha-floorplan#310] [Bugs] [answered]
Hi, I'm having problems with the "tap_action: more-info" function, it only shows me the first 4 sensors, the others don't.  Below is my code.  Thank you

**Comment by OzGav** [comment, marked as answer]
Check which layer those non working elements are in. They need to be on top of anything else so you can actually touch/click on them. If the pictures overlap that can also be a problem.
